public/js/controllers/contentController.js

```diff
@@ -4,17 +4,22 @@
  */
 angular.module('madisonApp.controllers')
   .controller('ContentController', ['$scope', '$stateParams', '$translate',
-    'pageService', 'SITE',
-    function ($scope, $stateParams, $translate, pageService, SITE) {
-      var page = $stateParams.page.replace(/-/g, '');
-      $translate('content.' + page + '.title', {title: SITE.name}).then(function(translation) {
-        pageService.setTitle(translation);
-      });
-      $translate('content.' + page + '.header', {title: SITE.name}).then(function(translation) {
-        $scope.header = translation;
-      });
-      $translate('content.' + page + '.body', {title: SITE.name}).then(function(translation) {
-        $scope.body = translation;
+    'pageService', 'SITE', 'Page', 'page',
+    function ($scope, $stateParams, $translate, pageService, SITE, Page, page) {
```

Comment:
`Page, page` has potential for all sorts of disaster.  Please use more verbose naming here.